docs/news.rst

```diff
@@ -86,21 +84,27 @@ New Features and Enhancements
 - ``CloseSpider`` and ``SpiderState`` extensions get disabled if no relevant
   setting is set (:issue:`1723`, :issue:`1725`).
 
+.. _`Code of Conduct`: https://github.com/scrapy/scrapy/blob/master/CODE_OF_CONDUCT.md
 
 API changes
 ~~~~~~~~~~~
 
-- Update form.py to improve existing capability PR #1137 (https://github.com/scrapy/scrapy/commit/786f62664b41f264bf4213a1ee3805774d82ed69)
-    Adds "formid" parameter for Form from_response()
-- `FormRequest.from_response` now allows to define through CSS selectors which
-  form from the response should be used. It previously supported only XPath
-  (:issue:`1382`).
-- Add ExecutionEngine.close() method #1423 (https://github.com/scrapy/scrapy/commit/caf2080b8095acd11de6018911025076ead23585)
+- ``FormRequest.from_response`` now allows: 
+  
+  - to set which Form you want to get through a new ``formid`` parameter (:issue:`1137`).
+
+  - to define through CSS selectors which form from the response should be used. 
+  It previously supported only XPath  (:issue:`1382`).
+
+- Add ExecutionEngine.close() method (:issue:`1423`).
+
     Adds a new method as a single entry point for shutting down the engine
-    and integrates it into Crawler.crawl() for graceful error handling during the crawling process.
+    and integrates it into Crawler.crawl() for graceful error handling during
+    the crawling process.
 
     TODO: explain what this does
```

Comment:
this TODO can be just removed -- explanation should be left for docstring or main docs